### DISCUSSION TITLE
How to generate impulse wav file for shairport-sync with Room EQ Wizard?

### DISCUSSION THREAD

**Discussion by jerryr** [mikebrady/shairport-sync#1287] [Q&A] [answered]
Hello All,
When exporting the impulse file from REW, what options should be selected?

Thanks for any help or pointers.

**Comment by mikebrady** [comment, marked as answer]
Did you have a look at the [wiki](https://github.com/mikebrady/shairport-sync/wiki/Digital-Signal-Processing-with-Shairport-Sync)?

**Reply by jerryr** [reply]
Don't know how I missed that.. need to get my eyes checked!
Thanks!
